common/meparser.cpp

```diff
@@ -73,7 +72,7 @@ USTATUS MeParser::parseMeRegionBody(const UModelIndex & index)
         return U_INVALID_ME_PARTITION_TABLE;
     }
     // Data partition always points to FPT header
-    if (meRegion.mid(ifwi16Header->DataPartition.Offset, sizeof(UINT32)) == FPT_HEADER_SIGNATURE) {
+    if (*(UINT32*)(meRegion.constData() + ifwi16Header->DataPartition.Offset) == FPT_HEADER_SIGNATURE) {
```

Comment:
Same here and below.

Reply:
Same as above, needs to be addressed in another PR, let's make an issue for it.